src/Entry.h

```diff
@@ -1,14 +1,20 @@
-//
-//  Entry.h
-//  LeapMusic
-//
-//  Created by Jonas Skog Andersen on 2015-02-06.
-//
-//
+#ifndef Entry_H
+#define Entry_H
```

Comment:
Becaus Entry_H is such a generic name, could I ask that this header guard is named LeapMusic_Entry_h?